interfaces/cython/cantera/lxcatml2yaml.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+# This file is part of Cantera. See License.txt in the top-level directory or
+# at https://cantera.org/license.txt for license and copyright information.
+
+"""
+lxcatml2yaml.py: Convert the LXCat integral cross-section data in XML format (LXCatML) to YAML format
+
+Usage:
+    lxcatml2yaml [--input=<filename>]
+                 [--database=<database name>]
+                 [--mech=<filename>]
+                 [--phase=<phase name>]
+                 [--insert]
+                 [--output=<filename>]
+
```

Comment:
Similar output will be automatically generated by `argparse` for a successfully installed console scipt (i.e. type `lxcat2yaml --help`).

It would be preferable if you describe purpose of this converter in a little more detail, as well as where data can be downloaded (or any other online references). Note that Sphinx should pull this docstring for Cantera documentation purposes; I provided instructions at the top.

Reply:
Fwiw, you should write this with Sphinx-generated output in mind. Examples are https://cantera.org/documentation/dev/sphinx/html/cython/yaml_conversion.html and https://cantera.org/documentation/dev/sphinx/html/yaml/yaml2ck.html